lib/recurly/client/adapter.rb

```diff
@@ -0,0 +1,46 @@
+require 'openssl'
+
+module Recurly
+  class Client
+    module NetHttpPersistentAdapter
+      protected
+      def configure_adapter(faraday)
+        faraday.adapter :net_http_persistent do |http| # yields Net::HTTP
+          # Let's not use the bundled cert in production yet
+          # but we will use these certs for any other staging or dev environment
+          unless BASE_URL.end_with?('.recurly.com')
+            http.ca_file = File.join(File.dirname(__FILE__), '../data/ca-certificates.crt')
+          end
+          http.open_timeout = 50
+          http.read_timeout = 60
+        end
+      end
+    end
+
+    module NetHttpAdapter
+      protected
+      def configure_adapter(http)
```

Comment:
realizing this probably should have been `faraday`

Reply:
https://github.com/recurly/recurly-client-ruby/commit/bb46c8869defe88df593d4e890d5a9de73713550